.github/workflows/push-to-ghcr.yml

```diff
@@ -0,0 +1,35 @@
+name: Push package to GHCR
+
+env:
+  DOTNET_VERSION: '6.0.x'
+
+on:
+  push:
+
+jobs:
+      push-store-image:
+        runs-on: ubuntu-latest
+        steps:
+          - name: 'Checkout GitHub Actions'
+            uses: actions/checkout@main
+
+          - name: Import GPG key
+            uses: crazy-max/ghaction-import-gpg@v6
+            with:
+              gpg_private_key: ${{ secrets.HELM_SIGN_PGP_PRIVATE_KEY }}
+              passphrase: ''
+
+          - name: Export PGP Private Key
+            run: |
+              gpg --export-secret-keys 'Helm Package Signing' > ~/.gnupg/private.gpg
+
+          - name: Helm Registry Login
+            run: helm registry login ghcr.io -u ${{ github.repository_owner }} --password ${{ secrets.GITHUB_TOKEN }}
+
+          - name: Package, Sign And Push To Registry
```

Comment:
You could maybe use a matrix here instead then you could have a yaml list and skip the for loop in the bash part to make it a bit easier to read https://code.dblock.org/2021/09/03/generating-task-matrix-by-looping-over-repo-files-with-github-actions.html

Reply:
Will have a look! :+1: